src/index/StringSortComparator.h

```diff
@@ -655,8 +657,9 @@ class TripleComponentComparator {
    * value according to the held locale
    */
   [[nodiscard]] SplitVal extractAndTransformComparable(
-      std::string_view a, const Level level, bool isExternal = false) const {
-    return extractComparable<SplitVal>(a, level, isExternal);
+      std::string_view a, const Level level, bool isExternal = false,
```

Comment:
Let us check if the `isExternal`and `inTextIndex` are actually used for the comparison (I think not), so maybe we can drop them completely from the splitVals.